src/Acuminator/Acuminator.Tests/Tests/StaticAnalysis/DacFieldAndReferencedFieldMismatch/Sources/MismatchedLengthDac.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using System.ComponentModel;
+using PX.Api;
+using PX.Data;
+
+namespace PX.Objects.HackathonDemo.DAC.InconsistentTypesOfDeclaredFieldAndReferencedDacFields
+{
+	[AttributeUsage(AttributeTargets.Property, AllowMultiple = false)]
+	public class StringKeyAttribute : PXDBStringAttribute
+	{
+		public StringKeyAttribute() : base(25) { }
+	}
+
+	[PXDBString(25)]
+	[PXString(12)]
+	[AttributeUsage(AttributeTargets.Property, AllowMultiple = false)]
+	public class AggregatedStringKeyAttribute : PXDBStringAttribute
```

Comment:
This and the next attribute are not derived from `PXAggregateAttribute`, therefore they are not aggregator attributes from the Acumatica platform's POV. The attributes declared on them won't be applied to DAC fields.

To make the attribute an aggregator it should look like this:
```c#
	public class AggregatedStringKeyAttribute : PXAggregateAttribute
```

Reply:
amended tests & code. you can take a look, hopefully everything's done now, @SENya1990

Reply:
@acasj you forgot about the documentation for the diagnostic.